### PR TITLE
Basic Setup

### PR DESCRIPTION
**Pull request by adlonymous**
Created basic setup and scripts as well as basic dressup route

### CODE REVIEW

File: .gitignore
Comment (distr0x) [inline .gitignore:1]: please add `node_modules` here so we don't commit that

Comment (adlonymous) [reply .gitignore:1]: ah mb, done it

Comment (distr0x) [reply .gitignore:1]: you also need to remove the `node_modules` folder that was already added 

`git rm -r --cached node_modules` should do the trick and then commit the changes, thanks!

**Comment by adlonymous** [comment]
removed node modules. Just need to finalize the different monke assets on each category (Background, Outfit, Sombrero, GIFs, Wallpapers, Banner, As Is)

**Comment by distr0x** [comment]
looks good! let's get this merged and start getting the assets in there

**Review by distr0x — APPROVED** [review]

**Pull request merged**